### PR TITLE
icon blog post

### PR DESCRIPTION
**Pull request by chrisdias**

### CODE REVIEW

File: blogs/2017/10/24/theicon.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+*We answered "orange".*
+
+## Apologies and Transparency
+
+I want to apologize for how long it has taken us to respond. We've been watching the feedback come in, we've read every single piece, and we've been having "lively" discussions on the next steps. We believe we have a rational plan and want to share it with you, and we look forward to a continuing dialog on what we now know is such a deeply controversial topic.
+
+For complete transparency, I want to share some history as we think it helps to understand how we arrived at orange in the first place.
+
+## Welcome to the family
```
Comment (octref) [inline blogs/2017/10/24/theicon.md]: Space after h2 for consistency and ensuring the same rendering.

File: blogs/2017/10/24/theicon.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+Until we landed on the current "fish", which tested well with admittedly, a small number of developers.
+
+## Rotation
+
+We have a lot more design freedom on the Mac (and Linux), which is why we have a different icon there. While we wanted to keep the same icon shape, we were not limited to a single color and we could resolve contrast issues by mounting the icon on a dark panel. The rotation actually adheres to the [Apple design guidelines](https://developer.apple.com/macos/human-interface-guidelines/icons-and-images/app-icon).
+
+That said, the feedback that the rotation is outdated and the mount makes the icon appear smaller than other icons in the Dock is completely fair.
+
+## Fatigue
```
Comment (octref) [inline blogs/2017/10/24/theicon.md]: Space after h2 for consistency and ensuring the same rendering.

File: blogs/2017/10/24/theicon.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+For folks that have difficulty seeing the difference between Stable and Insiders, the green and blue work better together. Here's the two icons using the Sim Daltonism app (Deuteranomaly):
+
+![Screen shot of Sim Daltonism app with the Deuteranomaly effect](2017_10_24_colorrecognition.png)
+
+It is inevitable that we will have multiple products that share colors, so we are no longer concerned that VSTS and VS Code share the blue color.
+
+We are [exploring using a non-tilted icon for Mac]( https://github.com/Microsoft/vscode/issues/36692) based on @donysukardi's comment here:  https://github.com/Microsoft/vscode/issues/6607#issuecomment-335664326. I can't say yet that we'll do this, but we are going to explore it and either way, we'll post updates in that issue.
+
+## Looking Ahead
```
Comment (octref) [inline blogs/2017/10/24/theicon.md]: Space after h2 for consistency and ensuring the same rendering.

File: blogs/2017/10/24/theicon.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+## Rotation
+
+We have a lot more design freedom on the Mac (and Linux), which is why we have a different icon there. While we wanted to keep the same icon shape, we were not limited to a single color and we could resolve contrast issues by mounting the icon on a dark panel. The rotation actually adheres to the [Apple design guidelines](https://developer.apple.com/macos/human-interface-guidelines/icons-and-images/app-icon).
+
+That said, the feedback that the rotation is outdated and the mount makes the icon appear smaller than other icons in the Dock is completely fair.
+
+## Fatigue
+By mid-summer, icon fatigue rolled in. We had made so many iterations, we finally said let's go with the current shape, and we checked in the "pre-release" design for Insiders. A pre-release is denoted by the hashed box on the right and bottom:
+
+![Pre-release icons in orange](2017_10_24_orangeprerelease.png)
```
Comment (Tyriar) [inline blogs/2017/10/24/theicon.md]: I don't think the images are linked to the right folder

Comment (gregvanl) [reply blogs/2017/10/24/theicon.md]: The blogs directory structure means the pages don't render images in GitHub or local view. The blog post will be on PPE shortly. 
Pine has a pending commit to fix the blog image problem.

**Pull request merged**